examples/guestbook-go/README.md

```diff
@@ -4,37 +4,38 @@ This example shows how to build a simple multi-tier web application using Kubern
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](../../docs/getting-started-guides):
-
-```shell
-$ cd kubernetes
-$ hack/dev-build-and-up.sh
-```
+This example assumes that you have a working cluster (see the  [Getting Started Guides](../../docs/getting-started-guides)).
+A GKE specific version of this tutoriual can be found at [https://cloud.google.com/container-engine/docs/tutorials/guestbook](https://cloud.google.com/container-engine/docs/tutorials/guestbook).
```

Comment:
We should stop using GCE and GKE abbreviation because we've already gotten feedback that its confusing and also because the guidelines say we should alway spell out Google Compute Engine and Google Container Engine.